apis/quay/v1/quayregistry_types.go

```diff
@@ -88,6 +94,13 @@ type Component struct {
 	// Managed indicates whether or not the Operator is responsible for the lifecycle of this component.
 	// Default is true.
 	Managed bool `json:"managed"`
+	// Overrides holds information regarding component specific configurations.
+	Overrides Override `json:"overrides,omitempty"`
```

Comment:
Have you seen how this show up in the yaml when empty? Something like `oc edit quayregistry/name` should suffice to make sure this is not showing up empty when `VolumeSize` is not set. I am asking because omitempty, in most of the cases, still shows empty structs (except when it is a pointer to a struct and it is nil).